src/handlers/handler.ts

```diff
@@ -11,3 +11,15 @@ export const helloWorld: Handler = (event: APIGatewayEvent, context: Context, ca
 
   callback(null, response);
 }
+
+export const helloUser: Handler = (event: any, context: Context, callback: Callback) => {
```

Comment:
Keeping the event type as `APIGatewayEvent` will actually work for our uses.  We're after `event.body.name`, which should be part of the definition for that type.